### PR TITLE
Common React Native components

### PR DESCRIPTION
**Pull request by skatshang**
This pull request added React_Native_Component.md, including the following changes:
1. 20 React Native components with explanations
2. Commonly used props of each components, with their explanations and accepted type

Future updates to consider:
1. Styling
2. Add link to official documentaion

Please review the changes and provide your feedback. Thanks!

### CODE REVIEW

**Comment by kevintan999** [comment]
Hey @skatshang, would you mind adding some of the official documentation that React Native has? i think some hyperlinks could work really well and point to more information.

**Comment by kevintan999** [comment]
And maybe add some type of introduction to introduce the reader on what this article is trying to teach?

**Comment by skatshang** [comment]
> Hey @skatshang, would you mind adding some of the official documentation that React Native has? i think some hyperlinks could work really well and point to more information.

**Comment by skatshang** [comment]
> Hey @skatshang, would you mind adding some of the official documentation that React Native has? i think some hyperlinks could work really well and point to more information.

Thanks for your advice! The links to the offical documentation have been added. They also helped me realize that some components I listed have been deprecated and should be removed from my introduction.

**Comment by skatshang** [comment]
> And maybe add some type of introduction to introduce the reader on what this article is trying to teach?

A summary has been added.

File: Topics/Tech_Stacks/React_Native_Component.md [hunk 1]
```diff
@@ -0,0 +1,295 @@
+# Common React Native Components
+
+## Summary
+
+This guide provides an overview of commonly-used React Native components for mobile app development.
```
Comment (kevintan999) [inline Topics/Tech_Stacks/React_Native_Component.md:5]: Summary looks good!

**Comment by kevintan999** [comment]
@skatshang maybe for some entries you can add a few code examples? that could help users to identify sample usage. Or add some customization tips to provide more insight.

File: Topics/Tech_Stacks/React_Native_Component.md (hunk 1 above)
Comment (skatshang) [reply Topics/Tech_Stacks/React_Native_Component.md:5]: Also added anchor links in Table of Contents under Summary, to help readers quickly jump between the sections.

**Comment by skatshang** [comment]
For the summary part, I also added anchor links for readers to quickly 

> @skatshang maybe add your page as an extension in Learning React Native. Or add some customization tips to provide more insight.

just added my document as a section in Learning React Native page. thanks!

**Comment by kevintan999** [comment]
The new changes are good, thanks for the good work! 

looks good to merge!

**Comment by kevintan999** [comment]
looks good to fix!

**Comment by skatshang** [comment]
> the links are not rendered on the website properly. please change the format.

Noted. will start a new PR to fix it

**Pull request merged**